lib/src/ui_elements/m_login_button_style.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+/// Style of the button.
+/// Use either [MLoginButtonStyle.blue] or [MLoginButtonStyle.white].
+class MLoginButtonStyle {
+  static const blue = MLoginButtonStyle._(
+    iconAsset: 'm_login_button_blue_icon.svg',
+    textColor: Color(0xFFFFFFFF),
+    buttonBackgroundAsset: 'm_login_button_blue_background.svg',
+    iconBackground: LinearGradient(
+      colors: [
+        Color(0xFFFFF0FC),
+        Color(0xFFE3FDF5),
+      ],
+    ),
+  );
+
+  static const white = MLoginButtonStyle._(
+    iconAsset: 'm_login_button_white_icon.svg',
+    textColor: Color(0xFF0065F5),
+    buttonBackgroundAsset: 'm_login_button_white_background.svg',
+    iconBackground: LinearGradient(
+      colors: [
+        Color(0xFF0069FF),
```

Comment:
Hier gibts eigentlich nicht den Grund wie in Android. Ist aber wahrscheinlich wegen "portability" einfacher hier auch die Werte hardcoded zu verwenden.